.github/workflows/forward_model_load_check.yml

```diff
@@ -30,6 +30,12 @@ jobs:
       - name: Build wheel
         shell: bash
         run: |
+          git clone --recursive https://github.com/VowpalWabbit/vowpal_wabbit.git
```

Comment:
you could merge this ci into this other one https://github.com/VowpalWabbit/vowpal_wabbit/blob/c7c14efc8e0e62a45ebc2917720823701a3ada44/.github/workflows/python_wheels.yml#L53

and re-use the built wheel

Reply:
This CI isn't building the most recent master's branch. I got it working here, think it makes sense to keep them separate